templates/container-instance-network-profile-subnet-delg.json

```diff
@@ -0,0 +1,127 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "networkProfileName": {
+      "type": "string",
+      "defaultValue": "aci-networkProfile",
+      "metadata": {
+        "description": "The name of the network profile"
+      }
+    },
+    "interfaceConfigName": {
+      "type": "string",
+      "defaultValue": "eth0",
+      "metadata": {
+        "description": "The name of the interface config"
+      }
+    },
+    "interfaceIpConfig": {
+      "type": "string",
+      "defaultValue": "ipConfig1",
+      "metadata": {
+        "description": "The name of the ip config for the interface"
+      }
+    },
+    "vnetName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the vNet that the container instance will connect to"
+      }
+    },
+      "vnetAddressSpaceCIDR": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the vNet that the container instance will connect to"
+      }
+    },
+    "subnetName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the subnet that the container instance will connect to"
+      }
+    },
+    "subnetAddressPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "The address prefix for the subnet that the container instance will connect to"
+      }
+    }
+  },
+  "variables": {
+    "templateBaseUrl": "https://raw.githubusercontent.com/DFE-Digital/bat-platform-building-blocks/master/templates/"
```

Comment:
I'm wondering it it would be an idea to have this as a parameter passed into the template to make development easier when you're on a branch. I can foresee some future person trying to make changes to this template on a branch and forgetting to update the `templateBaseUrl` and then wondering why changes aren't having an effect.
However if this is the convention in other templates (I've not checked) then I'll be happy to leave this as it is.

Reply:
This is standard for all templates.

Reply:
Its standard for the templates outside of the building block repo, but I've just had a look at all the templates in the building blocks repo and we don't seem to have any nested templates within nested templates like you're proposing here.

Reply:
Removed the linked template and `templateBaseUrl` variable